dll/win32/browseui/shellbars/CSearchBar.h

```diff
@@ -0,0 +1,149 @@
+/*
+ * ReactOS Explorer
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+
+#pragma once
+
+class CSearchBar :
+    public CComCoClass<CSearchBar, &CLSID_FileSearchBand>,
+    public CComObjectRootEx<CComMultiThreadModelNoCS>,
+    public IDeskBand,
+    public IObjectWithSite,
+    public IInputObject,
+    public IPersistStream,
+    public IOleCommandTarget,
+    public IServiceProvider,
+    public IBandNavigate,
+    public IWinEventHandler,
+    public INamespaceProxy,
+    public IDispatch,
+    public CDialogImpl<CSearchBar>
+{
+
+private:
+    // *** BaseBarSite information ***
+    CComPtr<IUnknown> pSite;
+    CComPtr<IAddressEditBox> fAddressEditBox;
```

Comment:
"field", perhaps?

Reply:
I copied that data member from `CAddressBand` and I believe that's what the `f` is short for.
The prefix `m_` appears to be more common, so that's probably what all the data members should be prefixed with instead.